### PR TITLE
Added MoorDyn and Npy packages

### PR DESCRIPTION
**Pull request by izabala123**

### CODE REVIEW

File: nests.json [hunk 1]
```diff
@@ -23,6 +23,22 @@
...
     },
+    {
+      "name": "MoorDyn",
+      "packages": [
+        "MoorDyn_v1",
+		"MoorDyn_v1_DLL",
+        "MoorDyn_v1_5",
+		"MoorDyn_v1_5_DLL",
+		"MoorDyn_v2",
+		"MoorDyn_v2_DLL"
```
Comment (klugier) [inline nests.json:34]: Do we need all versions? Why not keep only v2?

File: nests.json [hunk 2]
```diff
@@ -224,6 +252,18 @@
...
       "category": "core",
       "readme": "https://raw.githubusercontent.com/anboto/Xlnt/main/README.md"
+    },
+	{
+      "name": "Npy",
+      "packages": [
+        "Npy",
+		"plugin/libnpy"
+      ],
+      "description": "Npy / Npz library wrapper",
```
Comment (klugier) [inline nests.json:262]: Would be good to expand Npy and Npz shortcut.

File: nests.json (hunk 1 above)
Comment (izabala123) [reply nests.json:34]: I prefer to maintain them all as v1 is also used by commercial software and v1.5 has the same interface as v1, but internally uses v2. The pace of upgrading of this expensive commercial software is very slow, years late, I don't know why. And of course I want to support it.

File: nests.json (hunk 2 above)
Comment (izabala123) [reply nests.json:262]: Sorry, I don't understand you.

**Pull request merged**